prow/pj-on-kind.sh

```diff
@@ -31,7 +33,7 @@ function main() {
 
   # Deploy pod and watch.
   echo "Applying pod to the mkpod cluster. Configure kubectl for the mkpod cluster with:"
-  echo '>  export KUBECONFIG="$(kind get kubeconfig-path --name=mkpod)"'
+  echo ">  export KUBECONFIG='${KUBECONFIG}'"
```

Comment:
imo we should drop the `>`, but that's unrelated to this change.

Reply:
Shouldn't we update the version as well?
https://github.com/kubernetes/test-infra/blob/81a52fa245a4c2e5471b9ff59a579e791247037d/prow/pj-on-kind.sh#L80

Reply:
This should work with either, because we are specifying the same path.

0.6.0 didn't quite release yet. Given the team dev events today it will probably be tomorrow.

Reply:
https://github.com/kubernetes-sigs/kind/issues/1098 tracking, the code changes are done, but I'll need more time to write good release notes. we should be safe to merge this in the meantime and upgrade pj-on-kind in a follow-up.